pkgs/development/libraries/arpa2common/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+
```

Comment:
Why this extra new line?

Reply:
Used it to separate `native/buildInputs` from Nix's functions/lib. Should've put cmake after the line or removed the line entirely.